doc/user/command-line.xml

```diff
@@ -1678,7 +1718,7 @@ foo.c
 
         <para>
 
-        This then allows the user to
+        This allows you to
```

Comment:
are we consistently using you or the user for such? or are we all over the place?

Reply:
We're all over the place.  It was an attempt to see how it worked, after you posted that link to the Bazel doc standards. I'm not offended if we leave it alone.